x-pack/plugins/endpoint/server/types.ts

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+export interface EndpointData {
+  machine_id: string;
+  created_at: Date;
```

Comment:
What does `created_at` represent again? Why not just `@timestamp`?

Reply:
Date pretty much covers everything and it also handles UTC
https://www.elastic.co/guide/en/elasticsearch/reference/current/date.html